### DISCUSSION TITLE
Support forums in multiple languages

### DISCUSSION THREAD

**Discussion by phoerious** [community/community#2955] [Discussions]
Our GitHub organization is a purely English-language community at the moment. This excludes many people on this planet who are using our software. The whole point of GitHub Discussions is to provide a forum not only to developers and very tech-savvy people who are using GitHub anyway, but also to the normal end user. That makes it absolutely essential to have forums in different languages, otherwise we would have to run our own forum and then we have no need for GitHub Discussions.

The following points should be considered when implementing multi-lingual forums:

- maintainers may not speak the language, so we need community moderators,
- the default language displayed to users should be their browser/UI language, but it should be obvious that there are other languages to check as well,
- the category list shouldn't grow with quadratically with the number of languages and hence,
- the language setting should not be per category, but orthogonal to categories,
- category names should be translatable.

**Comment by shvchk** [comment]
You can somewhat overcome this limitation with sections. Like, English section, Deutsch section, Español section, etc.
